web/portal/platform/src/translations/ca.json

```diff
@@ -1,1054 +1,1055 @@
 {
-    "%s emulated": "%s emulat",
```

Comment:
Why is this part of this PR?

Reply:
Added thid row:
471 - "Impersonate as brand admin": "Impersonarse como admin de marca",